### ISSUE TITLE
Dataset Authorization Issue

### ISSUE THREAD

**Issue by ghost** [terraform-google-modules/terraform-google-bigquery#153]
My dataset authorization is getting removed by the dataset module.  It seems whatever access granted that is not part of access=[] clause are getting removed.

**Comment by alisha-sprinkle-inmar** [comment]
My team is experiencing the same issue. 
We use a google_bigquery_dataset_iam_binding within the same Terraform script. Terraform adds the service account mentioned in the binding to the access array on the first apply but promptly removes it on the second apply. The third apply puts the element back - so on and so forth.
We discovered this was because of the default value of the access array defined in the variables.tf. To resolve the issue we defined explicitly on the dataset an access array that contains both the default value of the module and the role/service account defined in the binding.  Now no changes are detected between subsequent applies. 
While this is more than likely intended behavior, it would have been nice to have had the default value and consequences of not explicitly stating the access variable on the module documented . It took eliminating other possible issues before realizing the access variable default was the culprit.
